src/TimeSteppers/TimeSteppers.jl

```diff
@@ -9,18 +9,15 @@ using GPUifyLoops: @launch, @loop, @unroll
 
 import Oceananigans: TimeStepper
 
-using Oceananigans: AbstractGrid
-
-using Oceananigans.SurfaceWaves: x_curl_Uˢ_cross_U, y_curl_Uˢ_cross_U, z_curl_Uˢ_cross_U,
-                                 ∂t_uˢ, ∂t_vˢ, ∂t_wˢ
-
 using Oceananigans.Architectures: @hascuda
 @hascuda using CUDAnative, CuArrays
 
 using Oceananigans.Architectures
+using Oceananigans.Grids
 using Oceananigans.Operators
 using Oceananigans.Coriolis
```

Comment:
Explicit imports might be nice here for all these physics terms... but we can save this for later.